src/bucket.ts

```diff
@@ -29,6 +28,7 @@ export interface BucketOptions {
 export default class Bucket {
   private client: KintoClientBase;
   public name: string;
+  private _endpoints: any;
```

Comment:
I'd prefer to avoid `any` if at all possible. You _should_ be able to use `KintoClientBase["endpoints"]` as the type.